Gitlab/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml

```diff
@@ -0,0 +1,149 @@
+variables:
+  SAM_TEMPLATE: {{cookiecutter.sam_template}}
+  AWS_ACCESS_KEY_ID_: ${AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY_: ${AWS_SECRET_ACCESS_KEY}
+  TESTING_STACK_NAME: {{cookiecutter.testing_stack_name}}
+  TESTING_REGION: {{cookiecutter.testing_region}}
+  TESTING_PIPELINE_EXECUTION_ROLE: {{cookiecutter.testing_pipeline_execution_role}}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.testing_cloudformation_execution_role}}
+  TESTING_ARTIFACTS_BUCKET: {{cookiecutter.testing_artifacts_bucket}}
+  TESTING_ECR_REPO: {{cookiecutter.testing_ecr_repo}}
```

Comment:
this should be optional, i.e., don't include if cookiecutter.testing_ecr_repo is empty

Reply:
Sure will update it!